Frequency/Board/Buildings/Barrack.py

```diff
@@ -0,0 +1,14 @@
+import pygame
+
+from Board.Buildings.Building2 import Building
+
+
+class Barracks(Building):
+    def __init__(self, player, tile):
+        self.Texture = [
```

Comment:
Op veel plaatsen hergebruiken we plaatjes. Maar we moeten kijken of het performance technische ook gewoon zo kan. Als ze niet als argument aan de constructor worden meegeven weet je zeker dat er niet mee geknoeid kan worden. Hernoem het wel even naar meervoud